src/Dapi/DapiApp.cs

```diff
@@ -199,6 +199,66 @@ public class DapiApp {
         /// </param>
         /// <param name="userInputs">
         /// built from the previous call's response, and the required user input.
+        /// </param>
+         public GetCategorizedTransactionsResponse getCategorizedTransactions(string accountID, DateTime fromDate, DateTime toDate, string accessToken, string userSecret) {
```

Comment:
Each endpoint should has a pair of methods, one for starting a new _operation_, and one for resuming it. This is the way the other methods' been done.